devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -211,13 +210,11 @@ def configure() {
 			zigbee.configureReporting(zigbee.RELATIVE_HUMIDITY_CLUSTER, 0x0000, DataType.UINT16, 60, 600, 1*100) +
 			zigbee.configureReporting(zigbee.TEMPERATURE_MEASUREMENT_CLUSTER, 0x0000, DataType.INT16, 60, 600, 0xA) +
 			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0020, DataType.UINT8, 30, 21600, 0x1)
-	} else if (isEWeLinkTh01()) {
+	} else if (isEWeLink()) {
 		return refresh() +
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0x104E]) +   // New firmware
-			zigbee.configureReporting(0xFC45, 0x0000, DataType.UINT16, 30, 3600, 100, ["mfgCode": 0xC2DF]) +   // Original firmware
-			zigbee.batteryConfig() +
-			zigbee.temperatureConfig(3600, 7200) +
-			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 3600, 7200, null)
+			zigbee.configureReporting(zigbee.POWER_CONFIGURATION_CLUSTER, 0x0021, DataType.UINT8, 3600, 7200, 0x10) +
+			zigbee.temperatureConfig(10, 7200, 50) +
+			zigbee.configureReporting(0x0405, 0x0000, DataType.UINT16, 10, 7200, 300)
```

Comment:
Why do You stop sending configuration for model Th01 ? Maybe You should split those configuration commands into 2 else-if sections ?

Reply:
I didn't stop sending the configuration for TH01, but the customer fed back some problems with the reported data, and we made some adjustments.